source/databases-collections.txt

```diff
@@ -1,13 +1,137 @@
 .. _vsce-databases-collections:
 
-==============================================
-Navigate Databases, Collections, and Documents
-==============================================
+==================
+Navigate Your Data
+==================
 
 .. default-domain:: mongodb
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
+
+Once you connect to your deployment using |vsce|, you can use the
+left navigation to:
+
+- Explore your databases, collections, and documents. 
+
+- Create new databases and collections.
+
+- Drop databases and collections.
+
+.. figure:: /images/dbs-colls-docs.png
+  :figwidth: 300px
+  :alt: Image showing deployment navigation
+
+Databases and Collections
+-------------------------
+
+When you expand an active connection, |vsce| shows the databases
+in that deployment. Click a database to view the collections that
+database.
+
+Create a New Database
+~~~~~~~~~~~~~~~~~~~~~
+
+To create a new database:
+
+1. Hover over the connection for the deployment where you want your
+   database to exist.
+
+#. Click the appearing :icon-fa5:`plus` icon.
+
+#. In the prompt, enter a name for your new database.
+
+#. Press the enter key.
+
+#. Enter a name for the first collection in your new database.
+
+   .. note::
+
+      You must initialize your new database by populating it with
+      a collection.
```

Comment:
This note really breaks up the last two numbered items because they're just so darn short. Do you think we could move this note up above the numbered instructions instead? I feel like it's important information to know before you start the procedure anyway.

Reply:
Agree. Moving above as you suggested.